src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -136,14 +137,40 @@ public void createDbScriptTable() {
         sqlScript.setType(COMMIT);
         sqlScript.setSchemaType(OWNER);
 
-        int exitCode = execute(sqlScript);
+        int exitCode = execute(sqlScript, false);
         tmpFile.delete();
         if (exitCode != EXIT_CODE_SUCCESS) {
             logger.error("Please execute script \"src/main/resources/create.sql\" manually");
             throw new ScriptExecException("Can't create DB objects used by SCMDB.");
         }
     }
 
+    public void executeCompileSchemas() {
```

Comment:
extract common code of `createDbScriptTable` and `executeCompileSchemas` and use it for both methods

Reply:
Done